tfhe-benchmark/benches/integer/vector_find.rs

```diff
@@ -0,0 +1,109 @@
+use benchmark::params_aliases::BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+use benchmark::utilities::{write_to_json, OperatorType};
+use criterion::{black_box, Criterion};
+use tfhe::integer::keycache::KEY_CACHE;
+use tfhe::integer::{IntegerKeyKind, RadixClientKey};
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::AtomicPatternParameters;
+use tfhe::MatchValues;
+
+pub fn match_value(c: &mut Criterion) {
+    let bench_name = "integer::match_value";
+
+    let param = BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+    let atomic_param: AtomicPatternParameters = param.into();
+    let param_name = param.name();
+
+    let (cpu_cks, cpu_sks) = KEY_CACHE.get_from_params(atomic_param, IntegerKeyKind::Radix);
+
+    let scenarios = vec![(64, 10, 32), (8, 256, 4)];
+
+    for (bits, num_elements, num_blocks) in scenarios {
+        let bench_id = format!("{bench_name}::{param_name}::{bits}bit_{num_elements}elements");
+
+        let cks = RadixClientKey::from((cpu_cks.clone(), num_blocks));
+
+        let mapping_data: Vec<(u64, u64)> = (0..num_elements as u64).map(|i| (i, i)).collect();
+        let match_values = MatchValues::new(mapping_data).expect("Invalid match values");
+
+        let input_val = 1u64;
+        let ct_input = cks.encrypt(input_val);
+
+        let mut group = c.benchmark_group(&bench_id);
```

Comment:
I believe that the group should be instantiated at the top of the function, right after `let bench_name = ...` . Otherwise, you may end up with veeeery long group name. You can look how group are handled in [integer benchmarks](https://github.com/zama-ai/tfhe-rs/blob/main/tfhe-benchmark/benches/integer/bench.rs#L46)